db/migrate/20140409134713_move_log_collection_depot_settings_to_file_depot.rb

```diff
@@ -5,20 +5,14 @@ class Authentication < ActiveRecord::Base
 
   class Configuration < ActiveRecord::Base
     serialize :settings, Hash
-    self.inheritance_column = :_type_disabled # disable STI
   end
 
-  class FileDepot < ActiveRecord::Base
-    self.inheritance_column = :_type_disabled # disable STI
```

Comment:
@lpichler When this migration was created, it would have been possible to run these migrations out of order if you were the author of the other migration.  Since we're long past this, I think it's fine to drop this as you have done... :+1: